src/ru/skypro/Main.java

```diff
@@ -3,6 +3,87 @@
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        byte clientOS = 1;
+        if ( clientOS == 0) {
+            System.out.println("Установите версию приложения для iOS по ссылке");
+        } else if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Андроид по ссылке");
+        }
+
+        byte clientOS2 = 1;
+        int clientDeviceYear = 2014;
+        if (clientOS2 == 0 && clientDeviceYear <= 2014) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        } else if (clientOS2 == 1 && clientDeviceYear <= 2014) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        } else {
+            System.out.println("Установите обычную версию");
+        }
+
+        int year = 1000;
+        if (( year % 4 == 0) && ((year % 100 != 0) || year % 400 == 0)) {
+            System.out.println(year + " високосный год");
+        } else {
+            System.out.println( year + " не является високосным годом");
+        }
+
+        int deliveryDistance = 99;
+        if (deliveryDistance < 20) {
+        System.out.println("Потребуется дней: 1 день");
+    } else {
+        if (deliveryDistance >= 20 && deliveryDistance < 60) {
+            System.out.println("Потребуется дней: 2 день");
+        } else {
+            if (deliveryDistance >= 60 && deliveryDistance < 100) {
+                System.out.println("Потребуется дней: 3 дня");
+            }
+        }
```

Comment:
Лучше объединить в единый оператор, тогда левые условия автоматически выполняются из-за невыполнения условия выше, можно упростить
```suggestion
        if (deliveryDistance < 20) {
        System.out.println("Потребуется дней: 1 день");
    } else if (deliveryDistance < 60) {
            System.out.println("Потребуется дней: 2 день");
        } else if (deliveryDistance < 100) {
                System.out.println("Потребуется дней: 3 дня");
            }
        }
```

Reply:
точно) можно же так было, спасибо